voting/src/contract.rs

```diff
@@ -84,14 +84,9 @@ pub fn stake_voting_tokens<S: Storage, A: Api, Q: Querier>(
 
     let key = &env.message.sender.as_slice();
 
-    let balance = Uint128::zero();
-
     let mut token_manager = match bank_read(&deps.storage).may_load(key)? {
         Some(token_manager) => Some(token_manager),
-        None => Some(TokenManager {
-            token_balance: balance,
-            participated_polls: Vec::new()
-        }),
+        None => Some(TokenManager::new()),
```

Comment:
You can add `#[derive(Default)]` to `TokenManager`, then this block becomes:

`let mut token_manager = bank_read(&deps.storage).may_load(key)?.unwrap_or_default();`

(I wanted to make that comment a few places on the master branch)